### ISSUE TITLE
Overfitting problem after dividing dataset

### ISSUE THREAD

**Issue by qianpiao** [rruisong/pytorch_federated_learning#10]
最近在用pytorch写FL代码时候，发现一个问题，一旦从MNIST或CIFAR10数据集中读取数据重新构建dataloader后，训练都会出现过拟合问题。具体表现为test loss下降数轮后，反而上升，而精度卡在一个不高的程度就不动了。哪怕如下图的代码，使用重新构建的train_loader_2在一个client上训练都会这样，而如果使用原装datasets.CIFAR10载入的train_dataset则不会出现过拟合，模型可以到达更高的精度

![QQ图片20241003203654](https://github.com/user-attachments/assets/043fb28a-288b-46c4-ae5c-ab3127b14a61)
我下载了您的代码，您的源代码里只打印了train loss，没打印test loss，而我打印了test loss发现也会出现先下降后上升的现象，如下图所示，请问有解决的办法吗？
![image](https://github.com/user-attachments/assets/7cce0127-fca9-495c-9215-00ac9ef5b90f)
